addons/pos_self_order/static/src/app/self_order_service.js

```diff
@@ -55,6 +55,8 @@ export class SelfOrder extends Reactive {
         this.ordering = false;
         this.orders = [];
         this.kitchenPrinters = [];
+        this.consultation = this.config.self_ordering_mode === "consultation" ||
```

Comment:
Why create `consultation` and not use `ordering` as before ?

Reply:
You're right, `ordering` as `false` means  we can't order so we are in consultation mode